packages/frontend/src/pages/Contribute.jsx

```diff
@@ -70,18 +113,110 @@ export default observer(() => {
       const canvas = document.getElementById('cosmo')
       canvas.width = width
       canvas.height = height
+      const renderer = new THREE.WebGLRenderer({ antialias: true, canvas })
 
-      const ctx = canvas.getContext('2d')
-      ctx.font = '2rem Azeret Mono'
-      ctx.fillStyle = 'white'
-      ctx.textAlign = 'center'
-      ctx.fillText(
-        "Bhargov's cosmo should be here.",
-        canvas.width / 2,
-        canvas.height / 2
+      // set camera
+      let camera = new THREE.PerspectiveCamera(
+        // fov
+        35,
+        // aspect
+        width / height,
+        // near
+        0.01,
+        // far
+        10000 * AU
       )
+      const defaultZoom = 1.7 * AU
+      camera.position.set(0, 0, defaultZoom)
+
+      const scene = new THREE.Scene()
+      let clock = new THREE.Clock()
+
+      // create particle buffers
+      const particleGeometry = new THREE.BufferGeometry()
+      let blackHoleCount = 0
+      let positions = new Float32Array(3 * N_plummer)
+      let mass = new Float32Array(N_particles).fill(1, 0, N_plummer)
+      particleGeometry.setAttribute(
+        'position',
+        new THREE.BufferAttribute(positions, 3)
+      )
+      particleGeometry.setAttribute('mass', new THREE.BufferAttribute(mass, 1))
+
+      // create plane
+      const planeGeometry = new THREE.PlaneGeometry(AU, AU, 8, 8)
+      const planeMaterial = new THREE.MeshBasicMaterial({
+        color: 0x049ef4,
+        wireframe: true,
+        transparent: true,
+        opacity: 0.1,
+        side: THREE.DoubleSide,
+      })
```

Comment:
I think we could adjust things here to make it look better, for example, make the plane larger, or change the color, etc. Maybe discuss with @ChialiT.
btw, must it be a `plane` and could not be a `sphere` or something else?

Reply:
yeah, i submit the new colors to @0xWOLAND

Reply:
I added the plane to help with orienting the user